config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224765345-10dd866c-9d57-4155-8a16-a5056d785877.png)

- [ ] It would be nice if you add a descriptive summary for your readme file to make your readme descriptive and professional :+1: